privacyidea/lib/policy.py

```diff
@@ -406,6 +405,7 @@ class REMOTE_USER(object):
     __doc__ = """The list of possible values for the remote_user policy."""
     DISABLE = "disable"
     ACTIVE = "allowed"
+    FORCE = "force"
```

Comment:
Currently there is no way of setting the action to `force` in the UI or did i miss something?
I couldn't really get it to work even when changing the value in the database...

Reply:
I was missing it in the policy definition.

Reply:
Setting UI in 458eb975